pkg/ddl/backfilling.go

```diff
@@ -431,8 +431,14 @@ func (w *backfillWorker) run(d *ddlCtx, bf backfiller, job *model.Job) {
 		})
 
 		// Change the batch size dynamically.
-		newBatchCnt := job.ReorgMeta.GetBatchSizeOrDefault(int(variable.GetDDLReorgBatchSize()))
-		w.GetCtx().batchCnt = newBatchCnt
+		currentBatchCnt := w.GetCtx().batchCnt
+		targetBatchSize := job.ReorgMeta.GetBatchSize()
+		if targetBatchSize != currentBatchCnt {
+			w.GetCtx().batchCnt = targetBatchSize
+			logger.Info("adjust ddl job config success",
+				zap.Int64("job ID", job.ID),
```

Comment:
```suggestion
				zap.Int64("jobID", job.ID),
```

We may need to file another PR to unify the logs from `job ID` to `jobID` for better log searching

Reply:
I will check the previous pr and if necessary modify it in this pr

Reply:
Check completed, no `job ID` string output to log in PR #57336